config/config.example.json

```diff
@@ -4,7 +4,8 @@
   "CLOUD_BUCKET": "see https://console.cloud.google.com/storage/browser?project=$GCLOUD_PROJECT",
   "CLIENT_ID": "see https://console.cloud.google.com/apis/credentials?project=$GCLOUD_PROJECT",
   "CLIENT_SECRET": "see https://console.cloud.google.com/apis/credentials?project=$GCLOUD_PROJECT",
-  "LIGHTHOUSE_SERVER": "your lighthouse server URL (optional)",
+  "PAGESPEED_SERVER": "your page speed server URL (optional)",
```

Comment:
How is the server URL used? Can we use https://www.npmjs.com/package/psi (which I think talks to https://developers.google.com/speed/docs/insights/v2/reference/pagespeedapi/runpagespeed).